docker-compose.yml

```diff
@@ -137,7 +152,7 @@ services:
       - HIKARI_CONNECTION_THREADS=16
       - RF_AUTHENTICATION_CACHE_ENABLE=true
       - BACKSPLASH_HISTOGRAM_CACHE_ENABLE=true
-      - BACKSPLASH_TILE_CACHE_ENABLE=true
+      - BACKSPLASH_TILE_CACHE_ENABLE=false
```

Comment:
Deliberate? I've also been turning it off because it's broken but haven't committed because I wasn't sure if we wanted dev to be broken until we fix caching

Reply:
Yeah, this was deliberate -- I think we need a task to either get the cache in order or just remove it at this point